packages/manager/apps/web/client/app/private-database/state/private-database-state.controller.js

```diff
@@ -66,9 +66,13 @@ export default class PrivateDatabaseStateCtrl {
       this.getOomList();
     }
 
-    this.privateDatabaseService.canOrderRam(this.productId).then((canOrder) => {
-      this.canOrderRam = canOrder;
-    });
+    this.privateDatabaseService
+      .getUpgradePlans(this.productId)
+      .then((response) => {
+        if (response.length > 0) {
+          this.canOrderRam = true;
+        }
+      });
```

Comment:
Code duplication
https://github.com/ovh/manager/pull/7899/files#diff-47f38362ec6da4c315e1c44e3508148b0c8ff06e9868c792243ccf4fa5d6f5a8R43
You should refactor this inside the private database service
```suggestion
    this.privateDatabaseService
      .canOrderRam(this.productId)
      .then((canOrderRam) => {
          this.canOrderRam = canOrderRam;
      });
```